### ISSUE TITLE
How to hide toolbar?

### ISSUE THREAD

**Issue by xbladesub** [qvacua/vimr#949]
I want to hide these bar from the left and from the right. Is it possible?
<img width="85" alt="Screen Shot 2022-05-28 at 1 00 58 PM" src="https://user-images.githubusercontent.com/45129790/170810826-1c8db08e-9c4c-4298-965a-38401035bbb0.png">

**Comment by AjayShanker-geek** [comment]
Yup, I managed to turn them off via `Preferences -> Tools -> ...`

**Comment by HerCerM** [comment]
There are also commands provided by VimR:

https://github.com/qvacua/vimr/wiki#commands-from-within-neovim

The relevant command to hide the tool buttons would be `:VimRHideToolButtons`
